src/Application/Adapters/WordPress/Preset.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NunoMaduro\PhpInsights\Application\Adapters\WordPress;
+
+use NunoMaduro\PhpInsights\Domain\Contracts\Preset as PresetContract;
+use NunoMaduro\PhpInsights\Domain\Insights\ForbiddenDefineFunctions;
+use NunoMaduro\PhpInsights\Domain\Insights\ForbiddenUsingGlobals;
+use PHP_CodeSniffer\Standards\Generic\Sniffs\PHP\ForbiddenFunctionsSniff;
+use SlevomatCodingStandard\Sniffs\Namespaces\AlphabeticallySortedUsesSniff;
+use SlevomatCodingStandard\Sniffs\TypeHints\DeclareStrictTypesSniff;
+use SlevomatCodingStandard\Sniffs\TypeHints\TypeHintDeclarationSniff;
+use Symfony\Component\Finder\Finder;
+
+/**
+ * @internal
+ */
+final class Preset implements PresetContract
+{
+    /**
+     * {@inheritDoc}
+     */
+    public static function getName(): string
+    {
+        return 'wordpress';
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public static function get(): array
+    {
+        return [
```

Comment:
On others Preset, we add a merge with DefaultPreset config. Can we have the same behaviour here ?

Reply:
Sure thing!